generators/chipyard/src/main/scala/System.scala

```diff
@@ -38,8 +38,5 @@ class System(implicit p: Parameters) extends Subsystem
 class SystemModule[+L <: System](_outer: L) extends SubsystemModuleImp(_outer)
   with HasRTCModuleImp
   with HasExtInterruptsModuleImp
-  with CanHaveMasterAXI4MemPortModuleImp
```

Comment:
Why is this deleted here? Am I missing something / where they are added?

Reply:
There's no imp trait anymore; it's been replaced with an InModuleBody call